freeipa/src/main/java/com/sequenceiq/freeipa/service/freeipa/user/model/UsersState.java

```diff
@@ -78,7 +88,12 @@ public void addWorkloadCredentials(String userName, WorkloadCredential creds) {
         }
 
         public UsersState build() {
-            return new UsersState(fmsGroups, fmsUsers, groupMembership, workloadCredentialMap);
+            return new UsersState(fmsGroups, fmsUsers, groupMembership, workloadCredentialMap, requestedWorkloadUsers);
+        }
+
+        public void addWorkloadUsername(FmsUser user) {
```

Comment:
nit: change the name to `addRequestedWorkloadUsername` or something to match the internal field name

Reply:
Fixed.